README.md

```diff
@@ -15,8 +15,8 @@ This plugin can be used in a variety of circumstances, for example if you have t
 
 ## Requirements
 
-Fastify 4.x.
-See [@fastify/http-proxy v7.x](https://github.com/fastify/fastify-http-proxy/tree/v7.x) for Fastify 3.x compatibility.
+Fastify 5.x.
```

Comment:
I think this approach is more readable:
https://github.com/Eomm/fastify-raw-body?tab=readme-ov-file#compatibility

WDYT?

Reply:
I don't disagree, I just wanted to follow the existing format to stay within scope of the issue I created. Rather than start to reformat _how_ the compatibility is shown.

But I do like this approach, rather than clicking (potentially several times) to find the plugin version that's compatible with a specific Fastify version.